src/node/BrowserFetcher.ts

```diff
@@ -178,18 +201,35 @@ export class BrowserFetcher {
     else assert(this._platform, 'Unsupported platform: ' + os.platform());
   }
 
+  /**
+   * @returns One of `mac`, `linux`, `win32` or `win64`.
+   */
   platform(): string {
     return this._platform;
   }
 
+  /**
+   * @returns One of `chrome` or `firefox`.
+   */
```

Comment:
we actually could update this return type to use the `Product` type (defined at the top of the file)